stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -324,31 +350,123 @@ Attribute convertScatterDimensionNumbers(vhlo::ScatterOpV1 vhloOp,
 
 #undef RETURN_CONVERTED_ENUM_ATTR
 
-/////////////////////////////////////
-/// VHLO --> StableHLO Operations ///
-/////////////////////////////////////
-
-struct VhloLegalizeToStablehloPass
-    : public impl::VhloLegalizeToStablehloPassBase<
-          VhloLegalizeToStablehloPass> {
-  void runOnOperation() override {
-    ConversionTarget target(getContext());
-    target.addIllegalDialect<vhlo::VhloDialect>();
-    target.addLegalDialect<stablehlo::StablehloDialect>();
-    target.addLegalDialect<func::FuncDialect>();
+template <typename... StringTy>
+void eraseAttrs(SmallVector<NamedAttribute>& attrs, StringTy... names) {
+  StringSet<llvm::MallocAllocator> nameSet({names...});
```

Comment:
If we use a set type that takes list initialization we could get away without a template function. From a callsite perspective the grouping makes sense:
`eraseAttrs(attrs, {things, to, erase})`

Reply:
Yeah, I considered that too, but I think it's nice to type fewer characters, and the template function doesn't look awfully complicated. Given that you approved the PR, I'll merge and then we can follow up on this if needed.